.github/workflows/github.yml

```diff
@@ -343,11 +364,6 @@ jobs:
     steps:
         - uses: actions/checkout@v4
 
-        - uses: actions/setup-python@v5
-          if: "${{ matrix.conan_profile != '' }}"
-          with:
-            python-version: '3.10'
```

Comment:
does Conan work fine now without this?

Reply:
Almost. Had to replace pip3 with pipx - requirement for newer Ubuntu, but both mac & ubuntu hosts work. Windows/msvc does not needs python.

Haven't checked mac os image, but ubuntu image comes with preinstalled python of roughly the same version.